spec/integration/spec.yml

```diff
@@ -66,3 +66,9 @@ spec:
     name: elasticsearch
     required: false
     $ref: "./elasticsearch/spec.yml"
+  - description: Errors configuration file
```

Comment:
Nit. Add a more complete description, and maybe mention that the file is only intended for package validation and should be ignored when installing or using the package.

That they are "errors" is more a Go implementation detail, but they are more like the validation result for a user.

Reply:
Updated description with more details.